src/TrailMap.ios.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
```

Comment:
Unless I messed something up, it should just be your code from develop with the tabs in it.  This is a pr from a feature branch to develop, so it should merge down.  I did a trial run to make sure it does a Ff merge.  If it looks off let me know and I'll fix it